### PR TITLE
Ready Playbook web for Swift!

### PR DESCRIPTION
**Pull request by jasoncypret**
**What does this PR do?** A clear and concise description with your runway ticket url.
Enables Swift docs for the public

**Screenshots:** Screenshots to visualize your addition/change
Screenshot 2023-12-21 at 2 00 06 PM

**How to test?** Steps to confirm the desired behavior:
4. See addition/change

#### Checklist:
- [ ] **LABELS** Add a label: `enhancement`, `bug`, `improvement`, `new kit`, `deprecated`, or `breaking`. See [Changelog & Labels](https://github.com/powerhome/playbook/wiki/Changelog-&-Labels) for details.
- [ ] **DEPLOY** I have added the `milano` label to show I'm ready for a review.
- [ ] **TESTS** I have added test coverage to my code.

### CODE REVIEW

File: playbook-website/app/components/hp_solutions_component.html.erb [hunk 1]
```diff
@@ -184,6 +184,42 @@
...
+							<%= link_to "/guides/getting_started/swift_setup" do %>
+								<%= pb_rails("card", props: {
+									padding: "xl",
+									shadow: "deeper",
+									border_radius: "lg",
+									cursor: "pointer",
+									hover: { scale: "sm" }
+								}) do %>
+									<%= pb_rails("image", props: {
+										alt: "HTML-CSS logo",
```
Comment (nidaqg) [inline playbook-website/app/components/hp_solutions_component.html.erb:204]:
```suggestion
										alt: "swift logo",
```

**Review by nidaqg — DISMISSED** [review]

**Review by nidaqg — APPROVED** [review]

**Comment by markdoeswork** [comment]
Swift is missing from the `User` `kit_category` Looks like the issue is with the env checking in these files too:

- `playbook-website/app/views/pages/kit_category_show.html.erb:13`
- `playbook-website/app/views/samples/show.html.erb:42`

https://pr2978.playbook.beta.hq.powerapp.cloud/kit_category/user?type=swift

swiftmissing

**Comment by nidaqg** [comment]
> Swift is missing from the `User` `kit_category` Looks like the issue is with the env checking in these files too:
> 
> * `playbook-website/app/views/pages/kit_category_show.html.erb:13`
> * `playbook-website/app/views/samples/show.html.erb:42`
> 
> https://pr2978.playbook.beta.hq.powerapp.cloud/kit_category/user?type=swift
> 
> swiftmissing

Good catch @markdoeswork ! Looks like swift docs are missing from all the category_show pages, not just `User`.

File: playbook-website/app/views/pages/kit_show.html.erb [hunk 2]
```diff
@@ -40,7 +40,7 @@
       <% if pb_doc_has_kit_type?(@kit, "react") %>
         <%= pb_rails("nav/item", props: { text: "React", link: kit_show_reacts_path(@kit), active: action_name == 'kit_show_react' }) %>
       <% end %>
-      <% if pb_doc_has_kit_type?(@kit, "swift") && Rails.env.development? %>
```
Comment (markdoeswork) [inline playbook-website/app/views/pages/kit_show.html.erb:43]: checkout this comment. There are more env checks to remove 

https://github.com/powerhome/playbook/pull/2978#issuecomment-1866911417

**Review by nidaqg — DISMISSED** [review]

File: playbook-website/app/views/samples/show.html.erb [hunk 3]
```diff
@@ -39,7 +39,7 @@
           <%= pb_rails("nav", props: { orientation: "horizontal", variant: "subtle" }) do %>
             <%= pb_rails("nav/item", props: { text: "Rails", link: "#{sample_show_path(@sample, type: :rails)}", active: current_sample_type.rails? }) %>
             <%= pb_rails("nav/item", props: { text: "React", link: "#{sample_show_path(@sample, type: :react)}", active: current_sample_type.react? }) %>
-            <%= pb_rails("nav/item", props: { text: "Swift", link: "#{sample_show_path(@sample, type: :swift)}", active: current_sample_type.swift? }) if Rails.env.development? %>
+            <%= pb_rails("nav/item", props: { text: "Swift", link: "#{sample_show_path(@sample, type: :swift)}", active: current_sample_type.swift? }) %>
```
Comment (jasoncypret) [inline playbook-website/app/views/samples/show.html.erb:42]: We don't have any swift samples. This causes a 404 when clicked. Can we just remove this whole line? I'm not sure why swift samples were created as we have no way of doing them.

Comment (nidaqg) [reply playbook-website/app/views/samples/show.html.erb:42]: Removing the line seems to be the best option, not sure when and why this was added for samples

**Review by nidaqg — APPROVED** [review]

**Pull request merged**